<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSonarsController</class>
 <widget class="QWidget" name="KSonarsController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QComboBox" name="SonarComboBox"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="checkTree">
         <column>
          <property name="text">
           <string>Available Graphical Elements</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>(Un) Check all</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <item>
           <property name="text">
            <string>Obstacles</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Path</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Target Coordinates</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
